src/pybind/mgr/influx/module.py

```diff
@@ -91,20 +106,75 @@ def can_run():
         else:
             return False, "influxdb python module not found"
 
+    @staticmethod
+    def get_timestamp():
+        return datetime.utcnow().isoformat() + 'Z'
+
+    @staticmethod
+    def chunk(l, n):
+        try:
+            while True:
+                xs = []
+                for _ in range(n):
+                    xs.append(next(l))
+                yield xs
+        except StopIteration:
+            yield xs
+
+    def queue_worker(self):
+        while True:
+            try:
+                points = self.queue.get()
+                if not points:
```

Comment:
prefer `points is None` to be more specific that `None` is the shutdown signal (as opposed to e.g. an empty chunk)

Reply:
Done!